ibu-imager/seedcreator/seedcreator.go

```diff
@@ -305,7 +305,7 @@ func (s *SeedCreator) createAndPushSeedImage() error {
 	if err != nil {
 		return fmt.Errorf("failed to query ostree status: %w", err)
 	}
-	if err = s.backupOstreeOrigin(statusRpmOstree); err != nil {
+	if err := s.backupOstreeOrigin(statusRpmOstree); err != nil {
```

Comment:
I didn't think the : was necessary here?

Reply:
lint in ci-job is failing on it

Reply:
It sort of right if you do in as oneliner than this err var is sort of new and exists only in context of this "if"

Reply:
Makes sense. Thanks!